pod/perldelta.pod

```diff
@@ -36,6 +36,62 @@ distinctions, this can be useful when serialising or encoding Perl data
 structures for export to other languages or data formats where such a
 distinction does matter.
 
+=head2 Support for not sharing all keys in large hash tables.
+
+This release includes support to build Perl so that it does not share
+all keys in hash tables as has been historically the case since Perl
+v5.6. The shared key behavior was added to Perl originally to support
+having many objects with the same keys without having to store each key
+many times. For objects that might be created hundreds or thousands of
+times this key sharing behavior can result in using significantly less
+memory. However key sharing comes at the cost of having to maintain an
+internal "master hash table" (PL_str_tab), and having to do two store
+operations per key stored in a hash table. It also can result in
+surprising consequences with programs that fork, causing the master hash
+table to be COWed (copy on write) into each forked processes memory
+space which can, in worse case, greatly increase the memory utilization
+of such scripts.
+
+The new behavior is an attempt to have our cake and eat it too. Small
+hash tables will continue to share keys, but once the hash table is
+used to store more than a certain number of keys (currently 64) it will
+not share the new keys that are added. This should benefit scripts that
+build large hash tables for aggregation purposes, especially those that
```

Comment:
I don't know what "for aggregation purposes" means, and in the context of this sentence I think it would be better to talk about mechanics - we want the feedback to tell us about the purposes that benefit or lose out.

Could we perhaps replace that phrase with "in which most keys are unique", or to spell it out "in which most keys are not repeated in any other hash"? I'd like some wording that would prompt a reader to think "oh, my 100M-key hash of seen values might benefit from that" since that's the use-case where I saw huge savings (without any forking).